zipline/testing/core.py

```diff
@@ -491,35 +491,33 @@ def write_daily_data(tempdir, sim_params, sids, trading_calendar):
 
 def create_data_portal(asset_finder, tempdir, sim_params, sids,
                        trading_calendar, adjustment_reader=None):
-    if sim_params.data_frequency == "daily":
-        daily_path = write_daily_data(tempdir, sim_params, sids,
-                                      trading_calendar)
-
-        equity_daily_reader = BcolzDailyBarReader(daily_path)
+    daily_path = write_daily_data(tempdir, sim_params, sids, trading_calendar)
```

Comment:
We always need daily data to do the `expected_shortfall` calculation. However, I added in a flag to turn off that calculation in most cases, so this change is actually no longer necessary for the current state of things. That being said, if anyone wanted to run an algo in a test minute mode that also wants to calculate `expected_shortfall`, this change would be necessary. I don't think it hurts to just always have daily data, but what do you think?